packages/trader/src/Modules/Trading/Components/Form/screen-large.jsx

```diff
@@ -7,27 +7,35 @@ import ContractType from '../../Containers/contract-type.jsx';
 import Purchase from '../../Containers/purchase.jsx';
 import TradeParams from '../../Containers/trade-params.jsx';
 
-const ScreenLarge = ({ is_market_closed, is_trade_enabled }) => (
-    <div
-        className={classNames('sidebar__items', {
-            'sidebar__items--market-closed': is_market_closed,
-        })}
-    >
-        {!is_trade_enabled ? (
-            <TradeParamsLoader speed={2} />
-        ) : (
-            <React.Fragment>
-                <Fieldset className='trade-container__fieldset trade-types'>
-                    <ContractType />
-                </Fieldset>
-                <TradeParams />
-                <div className='purchase-container'>
-                    <Purchase is_market_closed={is_market_closed} />
-                </div>
-            </React.Fragment>
-        )}
-    </div>
-);
+const ScreenLarge = ({ is_market_closed, is_trade_enabled }) => {
+    const [is_trade_enabled_state, setIsTradeEnabledState] = React.useState(false);
+
+    React.useEffect(() => {
+        setIsTradeEnabledState(is_trade_enabled);
```

Comment:
When `is_trade_enabled` prop is changed it should do the rerender.  Do we need to set state again to make it rerender?